status/boost_check_library.py

```diff
@@ -83,6 +107,33 @@ def check_organization_include(self):
                 ''',
                 'org-inc-one')
     
+    def check_organization_macros(self):
+        libdir = os.path.join(self.library_dir,'include','boost',self.library_name)
+        if os.path.isdir(libdir):
+            badmacros = {}
+            for root, dirs, files in os.walk(libdir):
+                if os.path.sep + 'detail' + os.path.sep in root or root.endswith(os.path.sep + 'detail'):
+                    continue
+                for item in files:
```

Comment:
why `item` and not `file` or `current_file`? I had to look this up when reading `os.path.join(root, item)`

Reply:
The natural thing to use here is `for file in files:`, but in python file is a builtin function. And `current_file` takes longer to type. But I'll fix it anyway.